bundle-workflow/python/assemble.py

```diff
@@ -0,0 +1,64 @@
+import os
+import sys
+import tempfile
+from assemble_workflow.bundle import Bundle
+from assemble_workflow.bundle_recorder import BundleRecorder
+from manifests.build_manifest import BuildManifest
+
+if (len(sys.argv) < 2):
+    print("Build an OpenSearch Bundle")
+    print("usage: assemble.sh /path/to/build_manifest")
+    exit(1)
+
+
+def get_tar_name(build):
+    return "-".join([build.name.lower(), build.version, 'linux', build.architecture]) + '.tar.gz'
+
+
+def get_plugins(build_components):
+    return [c for c in build_components if "plugins" in c.artifacts]
+
+
+def get_min_bundle(build_components):
+    return next(iter([c for c in build_components if "bundle" in c.artifacts]), None)
+
+
+build_manifest_path = sys.argv[1]
+build_manifest = BuildManifest.from_file(build_manifest_path)
+build = build_manifest.build
+artifacts_dir = os.path.dirname(os.path.realpath(build_manifest_path))
+output_dir = os.path.join(os.getcwd(), 'bundle')
+os.makedirs(output_dir, exist_ok=True)
+
+# Fetch built bundle components
+plugins = get_plugins(build_manifest.components)
+min_tar = get_min_bundle(build_manifest.components)
+tar_name = get_tar_name(build)
```

Comment:
Any reason why these shouldn't be properties of `BuildManifest` or (probably better) `Bundle`? This way you can do `bundle.install_plugins()` for example, too.